pinot-core/src/main/java/org/apache/pinot/core/plan/CombinePlanNode.java

```diff
@@ -73,87 +76,31 @@ public CombinePlanNode(List<PlanNode> planNodes, QueryContext queryContext, Exec
     _streamObserver = streamObserver;
   }
 
-  @SuppressWarnings({"rawtypes", "unchecked"})
   @Override
   public BaseCombineOperator run() {
-    int numPlanNodes = _planNodes.size();
-    List<Operator> operators = new ArrayList<>(numPlanNodes);
+    try (InvocationScope ignored = Tracing.getTracer().createScope(CombinePlanNode.class)) {
```

Comment:
record the number of children

Reply:
The API is not available yet. Will update once #8485 is merged